staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -134,22 +134,64 @@ if ($res) {
   Write-Host "Collecting Containerd running containers - tasks"
   & ctr.exe -n k8s.io t ls > "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
   $paths += "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
+
+  Write-Host "Collecting Containerd running containers - snapshot"
+  & ctr.exe -n k8s.io snapshot ls > "$ENV:TEMP\$timeStamp-containerd-snapshot.txt"
```

Comment:
did you test these two scripts in a AKS Windows node?
1. Download collect-windows-logs.ps1 to c:\k\debug to override the exist file and run it. Validate the collected log files and the content
2. Download loggenrator.ps1 to c:\k and update `isEnableLog` to `true` and validate the collected log files in C:\WindowsAzure\Logs\aks

Reply:
Yes, validate these two scripts both on docker runtime and containerd runtime.